src/Views/gagepage.html

```diff
@@ -240,20 +240,21 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.heatChartConfig">
                             <highchart id="chart2" config="vm.heatChartConfig"></highchart>
                         </div>
-                    </center> -->
+                    </center>
                     <center>
-                        <div ng-if="!vm.dischargeChartConfig" class="panel panel-default reportSpinner">
+                        <div ng-if="!vm.chartConfig" class="panel panel-default reportSpinner">
                             &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
                         </div>
-                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.dischargeChartConfig">
-                        <input type="checkbox" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()">
-                        <label for="logScaleDischarge" > Log Scale</label>
+                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
+
+                        <input type="checkbox" ng-if="vm.formattedDischargePeakDates.length > 0" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()" ng-hide="!data">
+                        <label for="logScaleDischarge" ng-if="vm.formattedDischargePeakDates.length > 0"> Log Scale</label>
                         
-                        <input type="radio" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')"> 
-                        <label for="ageData"> Age</label>
+                        <input type="radio"  ng-if="vm.measuredObj.control !== undefined" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')" checked> 
```

Comment:
Instead of adding ` ng-if="vm.measuredObj.control !== undefined"` to lines 253-257, you can wrap these lines in a div and put the ` ng-if="vm.measuredObj.control !== undefined"` in that div.

Reply:
I ended up keeping things like I had them since when I wrapped it in a div, it put it on a new line which I didn't like the look of. I have two different ng-if statements, and I want both on the same line on the website, so I just have it the long way.